tests/unit/HashArray/HashArrayWithoutDuplicatesTest.php

```diff
@@ -19,17 +20,10 @@
  */
 class HashArrayWithoutDuplicatesTest extends HashArrayTest {
 
-	public function constructorProvider() {
-		$argLists = [];
-
-		$argLists[] = [ HashArrayElement::getInstances() ];
-		$argLists[] = [ array_merge( HashArrayElement::getInstances(), HashArrayElement::getInstances() ) ];
```

Comment:
This is what I talked about in the commit message. These two constructors construct objects with the same elements. The second constructor call duplicates all elements, but because this is the "without duplicates" test, these duplicates are stripped on construction time. There are no assertions for this, so I came to the conclusion this second test case is redundant and can as well be removed.